lib/node-token-validation/src/cache/CacheTypes.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ValidCacheType } from "@azure/msal-common";
+
+/**
+ * Key value store for in-memory cache
+ * @public
+ */
+export type CacheKVStore = Record<string, ValidCacheType>;
```

Comment:
There's been some discussion about moving to `Map` instead of `Record` due to an attack vector with `Record`. With that in mind, I'd suggest using `Map` here instead to get ahead of that.

Reply:
Good call. Updated.